doc/source/whatsnew/v0.21.0.txt

```diff
@@ -307,8 +307,10 @@ Bug Fixes
 Conversion
 ^^^^^^^^^^
 
-- Bug in assignment against datetime-like data with ``int`` may incorrectly converte to datetime-like (:issue:`14145`)
+- Bug in assignment against datetime-like data with ``int`` may incorrectly convert to datetime-like (:issue:`14145`)
 - Bug in assignment against ``int64`` data with ``np.ndarray`` with ``float64`` dtype may keep ``int64`` dtype (:issue:`14001`)
+- Add a ``default`` argument to ``sys.getsizeof`` used in ``memory_usage`` to support PyPy. Objects on PyPy do not have a
```

Comment:
spelling, remove the word guess. remove reference to ``sys.getsizeof``, this is not user visible.